source/features/clean-repo-sidebar.tsx

```diff
@@ -31,8 +31,21 @@ async function cleanReleases(): Promise<void> {
 
 async function hideEmptyPackages(): Promise<void> {
 	const packagesCounter = await elementReady('.Layout-sidebar .BorderGrid-cell a[href*="/packages?"] .Counter', {waitForChildren: false});
-	if (packagesCounter && packagesCounter.textContent === '0') {
-		packagesCounter.closest('.BorderGrid-row')!.hidden = true;
+	if (!packagesCounter || packagesCounter.textContent !== '0') {
+		return;
+	}
+
+	packagesCounter.closest('.BorderGrid-row')!.hidden = true;
+
+	// Github seems to fetch internal packages separately, so if a repo only has internal packages, this makes the panel visible after they've loaded
+	const updatedPackagesCounter = await elementReady('.Layout-sidebar .BorderGrid-cell a[href*="/packages?"] .Counter:not([hidden="hidden"])', {
```

Comment:
I'm not very sold on this idea tbh but I'm not sure how to make it better, since I think any repo inside an Org can have internal packages?